### DISCUSSION TITLE
Tools /Export database /IO Error: Utility 'mysqldump.exe' not found in MySQL home 'MySQL Connector/Net'

### DISCUSSION THREAD

**Discussion by h2ku** [dbeaver/dbeaver#596] [Q&A] [answered]
Under mysql(mysql57) env on win10, 
executing Tools /Export database cause following error
_IO Error: Utility 'mysqldump.exe' not found in MySQL home 'MySQL Connector/Net'_

There was a workaround on the internet. [link](http://dba.stackexchange.com/questions/141219/dbio-error-utility-mysqldump-exe-not-found-in-mysql-home-mysql-connector-net)

I bet they changed location of the _mysqldump.exe_.

![error screen](https://cloud.githubusercontent.com/assets/7020454/16826556/34008ede-49b9-11e6-9c88-d451829c323f.jpg)

Thank you for the wonderful tool!

Best regards,
Hak Kim

**Comment by serge-rider** [comment]
"MySQL Connector/Net" isn't the best choice for client location (but it is in the registry and by default DBeaver sets first client from the registry).
But you can change location of MySQL client. (in the dropdown list on the connection properties page).
Or even set custom location (in the same dropdown).

Do you see any other solutions?

**Comment by h2ku** [comment]
Thank you for the info.

Why don't you change default client location to MySQL x.x?
That way 'export database' function would work.

**Comment by serge-rider** [comment, marked as answer]
Because there are too many different MySQL clients exists. 
Server, client tools, Workbench, connectors, etc. They all have different names which vary from version to version. And quite often there is no server installation on local machine.
After all I hope that it is not too hard to choose one manually..

**Comment by h2ku** [comment]
Noted. Thank you.
